packages/Table/src/Table.js

```diff
@@ -66,7 +66,19 @@ const Table = React.forwardRef((props, ref) => {
               {ColumnDefinitions.map((columnDefinition, columnIndex) => {
                 const position = { "data-row-index": rowIndex, "data-column-index": columnIndex };
 
-                const { cell, header, width, sticky, ...moreColumnProps } = columnDefinition.props;
+                const {
+                  cell,
+                  header,
+                  width,
+                  sticky,
+                  cellProps: _cellProps,
```

Comment:
Where does `_cellProps` come from? I don't understand this part 🙈 
Is the `_` a convention to indicate it might be undefined and that's why we check for the type later

Reply:
ohhh this is a javascript feature, when destructuring an object you can rename the variables:

ex:

```
const { name: renameThisToNewName, lastName } = user;

const name = "name is:" + renameThisToNewName // as example
```

Reply:
Oh nice, can't say I've used that feature when destructuring before. TIL 😅